bw2data/backends/proxies.py

```diff
@@ -477,6 +479,8 @@ def new_edge(self, **kwargs):
         exc = Exchange()
         exc.output = self.key
         for key in kwargs:
+            if key == "id":
+                raise ValueError("`id` must be created automatically")
```

Comment:
```suggestion
                raise ValueError(f"`id` must be created automatically, but id={kwargs["key"]} was given.")
```

Slightly easier to debug with this message. Same for the exception below.